db/init_db.js

```diff
@@ -20,6 +64,14 @@ async function buildTables() {
 async function populateInitialData() {
   try {
     // create useful starting data
+    const dummyProduct = {
+      name: "guitar",
+      price: 1000,
+      description: "This is a funky guitar",
+      category: "strings"
+    }
+    const product = await createProduct(dummyProduct)
+    console.log(product)
```

Comment:
If using this as a test output, add a message like "product created: " before it.

Reply:
Got it!